docs/content/preview/secure/enable-authentication/ysql-create-profiles.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Create login profiles
+headerTitle: Create and configure login profiles in YSQL
+linkTitle: Create login profiles
+description: Create and configure login profiles in YugabyteDB
+headcontent: Prevent brute force cracking with login profiles
+menu:
+  preview:
+    identifier: ysql-create-profiles
+    parent: enable-authentication
+    weight: 725
+type: docs
+---
+
+To enhance the security of your database, you can enable login profiles to lock accounts after a specified number of login attempts. This prevents brute force exploits.
+
+When enabled, database administrators with superuser (or in YugabyteDB Managed, `yb_db_admin`) privileges can create login profiles and assign the profiles to roles.
+
+When creating a profile, you must specify the number of failed attempts that are allowed before the account with the profile is locked.
+
+The number of failed attempts increments by one every time authentication fails during login. If the number of failed attempts is greater than the preset limit, then the account is locked.
```

Comment:
If the number of attempts **is equal to** the preset limit, then the account is locked.